src/main/java/org/apache/commons/lang3/CSVEscapeUtils.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.commons.lang3;
```

Comment:
Every file in Lang must include a license header.

Reply:
This is confusing IMO, we already have an Apache Commons CSV component, we should not mix in public CSV code in Lang IMO. If this is absolutely needed it should be package-private but I am doubtful.